source/_lovelace/button.markdown

```diff
@@ -11,10 +11,33 @@ The Button card allows you to add buttons to perform tasks.
 Screenshot of the Button card.
 </p>
 
-```yaml
-type: button
-entity: light.living_room
-```
+## Visual Editor
+
+<p class='img'>
+<img src='https://i.imgur.com/HJybKBf.jpg' alt='Screenshot of the button card Visual Editor'>Screenshot of the Button card Visual Editor</p>
+
+### Configuration
+
+{% configuration_basic %}
+Entity:
+  description: Choose a Home Assistant entity from the dropdown menu
+Name:
+  description: Enter a name for the button. The default is the entity name.
```

Comment:
Instead of an action. Describe the field instead.


Something to the degree of
```
The Name that will show on the button
```